modules/ROOT/pages/munit-cookbook.adoc

```diff
@@ -11,5 +11,8 @@ This section links to examples that demonstrate common MUnit usages.
 | xref:mock-before-after-foreach-cookbook.adoc[Mocking a Message Before and Inside a Foreach Processor] | Learn how to mock a processor that can return different payloads.
 | xref:test-mock-errors-cookbook.adoc[Testing and Mocking Errors] | Learn how to mock expected errors in your code.
 | xref:utility-classes-cookbook.adoc[Using Utility Classes From a Test] | Learn how to use custom Java classes in you MUnit tests.
+| xref:test-flow-with-scheduler-cookbook.adoc[Testing a Flow with Scheduler] | Learn how to test a flow that is triggered by an scheduler.
```

Comment:
Should this [Testing...] text be the same as that in the nav.adoc? They are a bit different.